src/components/ProfileGrades.tsx

```diff
@@ -0,0 +1,42 @@
+import Chart from "react-apexcharts"
+import { useState } from "react"
+import { AiFillCaretLeft, AiFillCaretRight } from "react-icons/ai"
+import { gradeChartOptions } from "~utils/styling";
+
+export interface GradeDistribution {
+  name: string;
+  series: ApexAxisChartSeries;
+}
+
+export const ProfileGrades = ({ gradeDistributionData } : { gradeDistributionData: GradeDistribution[] }) => {
+  const [page, setPage] = useState(0);
+
```

Comment:
I know its picky, but use === when checking js conditionals to avoid possible misuse later